Library/Formula/open-mpi.rb

```diff
@@ -27,7 +29,7 @@ class OpenMpi < Formula
   conflicts_with "lcdf-typetools", :because => "both install same set of binaries."
 
   depends_on :java => :build
-  depends_on :fortran => :recommended
+  depends_on :fortran if build.with?("fortran")
```

Comment:
I would to fix the problem in the core since we repeatedly discourage others to do this.